sentences/el/homeassistant_HassGetState.yaml

```diff
@@ -1,4 +1,39 @@
 language: el
 intents:
   HassGetState:
-    data: []
+    data:
+      - sentences:
+          # - (do you know|tell me|<what_is>) [the state of] <name> [in <area>]
+          - (ποι(ός|α|ο|οί|ές|α) είναι) [κατάσταση] <name> [<area>]
```

Comment:
Just noticed a minor typo, it should be: (ποι(ος|α|ο|οι|ες). The respective grammar rule defines in that case that these words shall be not emphasised (EL: 'δεν τονίζονται').

Reply:
Oh, true, elementary mistake :smile: I'll fix it.